Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -448,6 +448,7 @@ extension BrowserViewController: WKUIDelegate {
     }
 
     func webView(webView: WKWebView, runJavaScriptAlertPanelWithMessage message: String, initiatedByFrame frame: WKFrameInfo, completionHandler: () -> Void) {
+        self.tabManager.selectTab( self.tabManager.getTab(webView) )
```

Comment:
I will remove these, on your request, although it makes readability easier for me.  I use it when there is multiple nested function calls with arguments, although not necessary for the first level ie getTab(webView).